records_mover/records/sources/table.py

```diff
@@ -33,6 +35,7 @@ def __init__(self,
         self.schema_name = schema_name
         self.table_name = table_name
         self.driver = driver
+        # TODO: use this throughout
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker